### DISCUSSION TITLE
Security onion 2.4.80 download

### DISCUSSION THREAD

**Discussion by SliuzasLukas** [Security-Onion-Solutions/securityonion#13370] [Unsupported Versions] [answered]
Hi,

I am a bit confused. Is the 2.4.80 out yet or not?

When I click the link, I get this error message:

{
    "code": "not_found",
    "message": "File with such name does not exist.",
    "status": 404
}

**Comment by defensivedepth** [comment]
Where are you attempting to download from?

You can download from here: https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md

**Reply by SliuzasLukas** [reply]
I am using this https://github.com/Security-Onion-Solutions/securityonion/releases/tag/2.4.80-20240625

![image](https://github.com/user-attachments/assets/92e94c96-7765-4a2f-bb05-3bd237e7261c)

**Reply by defensivedepth** [reply, marked as answer]
This will always have the latest link + verification info.


![image](https://github.com/user-attachments/assets/d99f85a3-7ae5-4af2-892f-834e5b36d187)

**Reply by SliuzasLukas** [reply]
Issue was with the link in the release. Release date should be 20240624 instead of 20240625

**Reply by TOoSmOotH** [reply]
I updated the tag and release. Should be good to go.
